src/components/index.tsx

```diff
@@ -126,18 +133,58 @@ const getSizes = () => {
   };
 };
 
+async function submitCustomRUM(
+  payloadType: string,
+  payloadSchemaVersion: number,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  payload: any
```

Comment:
Just a nitpick, works perfectly fine as is..

But if you would put the `payloadSchemaVersion` as last argument here and set a default, calls to this function would look nicer, (as in omitting the `payloadSchemaVersion`)

Also IMO it should be possible to not use `any` but `Record<string, never>`. Best would be to create an interface for this.

Reply:
Also, why have already 2 versions? I've seen both `1` and `2`

Reply:
> Also IMO it should be possible to not use any but Record<string, never>. Best would be to create an interface for this.

That gives me some trouble:

![image](https://user-images.githubusercontent.com/273727/97696543-05da0680-1aa6-11eb-96b0-8c7463b88ecd.png)

Reply:
This is also only guessed. Its possible it could work with `Record<string, string | number>`.

But: This strengthens my opinion that the payload should have a dedicated interface.

Reply:
I'll do `Record<string, string | number | object>` then.